app/client-app/src/components/shared/SurveyPage.jsx

```diff
@@ -171,8 +194,9 @@ const SurveyPage = ({
             >
               Clear Response
             </Button>
+            <WebhookCounter />
```

Comment:
have this conditionally render on when `webhookCount != null`? that should fulfill it only showing in preview mode (which passes a numeric count) vs  real survey (which doesn't)?